setup_scripts/moab_setup.sh

```diff
@@ -9,16 +9,22 @@ mkdir bld
 MOAB_INSTALL_DIR=$INSTALL_ROOT/MOAB/install
 mkdir $MOAB_INSTALL_DIR
 git clone https://bitbucket.org/fathomteam/moab
+
 cd moab
-git checkout Version5.0
```

Comment:
The switch to `cmake` makes sense, but why are we not using Version5.0 anymore? I can't recall...

Reply:
The reason for this is that Version5.0 has no easy means of disabling BLAS or LAPACK in the installation. In order to install without either - the purpose of a lot of these changes - I'd either need to modify Version5.0, or just dodge it entirely. The latter was easier, but I admit I don't know much about the functional differences between that and the master branch,

Reply:
Gotcha, that makes sense. Thanks!